packages/stryker/src/Sandbox.ts

```diff
@@ -44,8 +44,8 @@ export default class Sandbox {
     return sandbox.initialize().then(() => sandbox);
   }
 
-  public run(timeout: number, testHooks: string | undefined): Promise<RunResult> {
-    return this.testRunner.run({ timeout, testHooks });
+  public run(timeout: number, testHooks: string | undefined, mutant?: TestableMutant): Promise<RunResult> {
+    return this.testRunner.run({ timeout, testHooks, ...(mutant && { mutatedFileName: this.fileMap[mutant.fileName]}) });
```

Comment:
Could we move the locating of the mutated file to the `runMutant` call? It is specific to the case of running when targeting a mutant. I feel like the separation of concerns is intermingled now.